<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="MacBook-Pro.local" failures="1" tests="7" name="Test" time="39.743" errors="0" timestamp="2024-12-08T23:07:47 EST">
  <testcase classname="com.testCases.TC_001_AccountCreationTest" name="test_account_registration" time="0.839"/>
  <testcase classname="com.testCases.TC_002_LoginTest" name="verify_login" time="0.632"/>
  <testcase classname="com.testCases.TC_004_LoginDDT_Fail_Scenarios" name="verify_loginDDT" time="0.584"/>
  <testcase classname="com.testCases.TC_004_LoginDDT_Fail_Scenarios" name="verify_loginDDT" time="0.464"/>
  <testcase classname="com.testCases.TC_004_LoginDDT_Fail_Scenarios" name="verify_loginDDT" time="10.338"/>
  <testcase classname="com.testCases.TC_004_LoginDDT_Fail_Scenarios" name="verify_loginDDT" time="10.384"/>
  <testcase classname="com.testCases.TC_004_LoginDDT_Fail_Scenarios" name="verify_loginDDT" time="10.383">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.testCases.TC_004_LoginDDT_Fail_Scenarios.verify_loginDDT(TC_004_LoginDDT_Fail_Scenarios.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="com.testCases.TC_003_LoginDDT" name="verify_loginDDT" time="0.0">
    <ignored/>
  </testcase> <!-- verify_loginDDT -->
</testsuite> <!-- Test -->
